packages/backend/src/api.ts

```diff
@@ -36,6 +42,7 @@ export function createAPIApp(): express.Application {
     next();
   });
   app.use(traceAPICall);
+  app.use(validateRequest(routeValidator));
```

Comment:
Any preference on having this as a middleware vs. a function wrapper like `authenticateRequest`? I figured this way validation happens across the board without having to think about it. My only concern is it is too much indirection and would look _magical_ to a new dev on the project?